### PR TITLE
feat: implement CallCallablePoint and ValidateDynamicLinkInterface to cosmwasmAPI

### PR DESCRIPTION
**Pull request by loloicci**
## Description

solves https://github.com/line/cosmwasm/issues/273

This PR

- updates depending wasmvm
  - modify tests for it
- adds new functions generate events from callable points
- implements CallCallablePoint to cosmwasmAPI
- implements ValidateDynamicLinkInterface to cosmwasmAPI
- adds tests for new APIs

## Motivation and context

see https://github.com/line/cosmwasm/issues/273

## How has this been tested?

unit tests in x/wasm/keeper/api_test.go
integrate tests in x/wasm/dynamic_link_test.go

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly. (not needed)
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml` (not needed)

### CODE REVIEW

File: x/wasm/keeper/api.go [hunk 1]
```diff
@@ -1,7 +1,10 @@
 package keeper
 
 import (
+	"fmt"
+
 	sdk "github.com/line/lbm-sdk/types"
+
 	wasmvm "github.com/line/wasmvm"
 	wasmvmtypes "github.com/line/wasmvm/types"
```
Comment (dudong2) [inline x/wasm/keeper/api.go:9]:
```suggestion
	"fmt"

	sdk "github.com/line/lbm-sdk/types"
	wasmvm "github.com/line/wasmvm"
	wasmvmtypes "github.com/line/wasmvm/types"
```

The order of the imports is golang's native modules, external modules, github.com/line/xxx, github.com/line/wasmd/xxx.

**Review by dudong2 — APPROVED** [review]

File: x/wasm/keeper/api.go (hunk 1 above)
Comment (dudong2) [reply x/wasm/keeper/api.go:9]: This isn't a big deal, though, so you can merge as is if other people think it's okay.

File: x/wasm/dynamic_link_test.go [hunk 2]
```diff
@@ -477,4 +477,5 @@ func TestDynamicCallCalleeFails(t *testing.T) {
 	}
 	res, err = h(data.ctx, &executeMsg)
 	assert.ErrorContains(t, err, "Error in dynamic link")
+	assert.ErrorContains(t, err, "RuntimeError: unreachable")
```
Comment (da1suk8) [inline x/wasm/dynamic_link_test.go:480]: You added this to make sure it is panic!(), right?

Comment (loloicci) [reply x/wasm/dynamic_link_test.go:480]: Yes. Because it is a test calling `do_panic` and it is generated `panic!()` in rust.

**Review by da1suk8 — APPROVED** [review]

**Review by dudong2 — APPROVED** [review]

**Pull request merged**